parcels/fieldset.py

```diff
@@ -1326,10 +1326,10 @@ def from_modulefile(cls, filename, modulename="create_fieldset", **kwargs):
         spec.loader.exec_module(fieldset_module)
 
         if not hasattr(fieldset_module, modulename):
-            raise IOError(f"{filename} does not contain a {modulename} function")
+            raise OSError(f"{filename} does not contain a {modulename} function")
```

Comment:
Is this really an OSError? Perhaps better a ValueError?

Reply:
This was just an update by pyupgrade. I'd prefer to give attention to all the errors in one go (as there are a lot of assertion errors that need to turn into ValueErrors)